debian/bionic/foreman/control

```diff
@@ -100,20 +100,6 @@ Description: metapackage providing libvirt dependencies for Foreman
  .
  The foreman package should be installed along with this package.
 
-Package: foreman-mysql2
-Architecture: all
-Section: web
-Priority: extra
-Replaces: foreman-mysql
```

Comment:
Should we add `Replaces: foreman-mysql2` on the main foreman package to make sure it gets removed on upgrades?

Reply:
Added.